README.md

```diff
@@ -26,6 +26,7 @@ or working on a new complex site feature in Go) we can consider providing a sepa
 This will allow you to run Preview Deploys on Netlify so you (and the rest of the Community) can browse your version 
 before deploying it to `master`.
 
+To know more about it do check this link :[https://docs.suitecrm.com/user/advanced-modules/workflow/#_creating_a_workflow_process]
```

Comment:
I think this is the wrong link, that page is for creating a SuiteCRM workflow while this README section is referring to a GitHub contribution workflow. It looks like the correct link is most likely: 

https://docs.suitecrm.com/community/contributing-to-docs/

But it looks like this has already been referenced under the ```## Contributing``` section further up.